addons/website_sale/controllers/main.py

```diff
@@ -621,6 +621,11 @@ def _get_vat_validation_fields(self, data):
         return {
             'vat': data['vat'],
             'country_id': int(data['country_id']) if data.get('country_id') else False,
+            # To make sure VIES check can be triggered.
+            'is_company': True,
```

Comment:
I'm not convinced by that. This goes against this commit: https://github.com/odoo/odoo/commit/b2257dd0b38f1cd6a0ab93b6f57fb3628c2ce5fb#diff-1ef5fe83935572c9b9b9e6eec69a44551023f3b92fe38bed4f765a24e16694b4

Reply:
Hi @chevalierv ,
its not that we need to check individual, the problem, is no matter what is going through website (company or individual), it is treated as individual. So what it ends up doing, is VIES is never checked, because it is only checked for companies (in backend).

And in controller, dummy partner is created which is always is_company=False. It does not try to determine whether entered data is for company or individual.

If you say it goes against that commit, then there should be a logic that you can identify if address is entered for company or individual. Otherwise VIES will never be called:)

Reply:
Hey,

I've checked with @paju-odoo and it is the expected behavior. In E-commerce, we consider our flows to be always B2C.

Reply:
Thats weird. Whats the point in allowing to enter company name and VAT then?:)

Reply:
Hi @paju-odoo yes I am referring validating that field (on address form). Currently that field only does basic validation, using VAT rules, but it it does not check whether that VAT is valid according to EU VIES system. VAT can be valid according to basic rules, but it can be invalid according to VIES (e.g. company no longer exists). 

And VIES check does not work, because of this change: https://github.com/odoo/odoo/commit/b2257dd0b38f1cd6a0ab93b6f57fb3628c2ce5fb#diff-1ef5fe83935572c9b9b9e6eec69a44551023f3b92fe38bed4f765a24e16694b4

Not sure why it was done? I think it makes sense to validate via commercial partner? Now because dummy partner is created on the fly when address form is submitted, it is created with `is_company=False`, thus never calling VIES.

P.S. Also I improved error message, to include partner name, because currently it always show this weird `False` as a name..:)

Reply:
I think we should improve the partner creation in the future versions of Odoo, by checking if the partner is an individual or a company, but I don't see the added value of checking the VIES in older versions, since this field does not make sense on individuals anyway...

Reply:
Unfortunately no, it can only be done in the back-end. If we decide to change the address form in the checkout process at some point, that will be improved but this is no priority